src/app/infinitescroll/infinitescroll.directive.ts

```diff
@@ -42,20 +42,22 @@ export class InfiniteScrollDirective implements OnInit {
     });
   }
 
-  @HostListener('scroll', ['$event']) 
+  @HostListener('scroll', ['$event'])
   onScrollContainer(event: any) {
     if (this.element.scrollHeight > this.lastCheckedHeight) {
       this.previousScrollHeight = this.element.scrollHeight;
-      let remainingHeight = this.element.scrollHeight - (this.element.offsetHeight + this.element.scrollTop);
+      let remainingHeight = this.element.scrollHeight
+        - (this.element.offsetHeight + this.element.scrollTop);
       let remainingElement = Math.ceil(remainingHeight / this.eachElementHeightInPx);
       if (remainingElement < this.fetchThreshold) {
         this.lastCheckedHeight = this.element.scrollHeight;
         this.fetchMore.emit();
       }
-    } else if (this.element.scrollHeight < this.previousScrollHeight) { // To check if the list is reloaded
+      // To check if the list is reloaded
```

Comment:
By moving the comment out of the block you are saying that it now belongs in the block above it. Is this your intention?

Reply:
I was getting linter error because it crosses 100 characters limit in a line. So that I move it to next line.

Reply:
explaining it here won't help everyone else that reads it, you need to make the code right

Reply:
I misread this one, disregard.